tsconfig.json

```diff
@@ -6,7 +6,8 @@
       "outDir": "./dist",
       "strict": true,
       "lib": [
-        "es5"
+        "es5",
+        "dom"
```

Comment:
Why is DOM here? This is a generic library, so shouldn't require the web-browser DOM.

Reply:
<img width="612" alt="Screen Shot 2022-01-31 at 12 31 09 PM" src="https://user-images.githubusercontent.com/4365774/151752394-e65785c6-e6fa-4b4d-8df4-9e2ce5b1dcfc.png">

^ was getting these typescript errors without it, stackoverflow solution suggested adding dom to fix it but good call out, I just ts-ingored those lined